doc/settings.md

```diff
@@ -48,6 +50,23 @@ files from disk by specifying an `defaultCharset` key. The default is
     'defaultCharset': 'UTF-8'
 ```
 
+## relocateBuildTree
+That is, real object, library or exec directories are relocated to the
```

Comment:
I don't understand the sentence above

Reply:
This is the description of this given in grpbuild documentation 
https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug/companion_tools.html

Reply:
it only makes sense if the previous sentence is present "With this option it is possible to achieve out-of-tree build."